### DISCUSSION TITLE
【灌水】生吃了三个西红柿胃痛呜呜呜

### DISCUSSION THREAD

**Discussion by ZheLUE** [share121/inter-knot#434] [其他]
四点清完每日看见冰箱还有西红柿，当早餐吃了，5：42就开始痛了呜呜呜
![搜图神器_1715359578671](https://github.com/user-attachments/assets/31e2cdd5-54a4-4329-9753-eca90f82a2df)

**Comment by SaKura-071721** [comment]
有实力，建议吃点健胃消食片。
还有，我老婆真好看。嘿嘿嘿嘿

**Comment by warning-Kirov-reporting** [comment]
已经过去 4 小时了，不知到发帖人怎么样了

**Comment by ZheLUE** [comment]
在腹泻 想死

**Reply by SaKura-071721** [reply]
正常情况，多喝热水窜窜就好了。哈哈哈哈哈哈

**Comment by xinxingjie** [comment]
冰镇tamato当早饭吃，你也想得到阿哈的瞥视吗[狗头]希望人没逝
